src/_utils/crm-web-api.ts

```diff
@@ -34,13 +34,19 @@ export const CRMWebAPI = {
     return value;
   },
 
-  parseErrorMessage: async function (json) {
-    if (json && json.error) return json.error;
-
+  parseErrorMessage: function (json) {
+    if (json) {
+      if (json.error) {
+        return json.error;
+      }
+      if (json._error) {
+        return json._error;
+      }
+    }
```

Comment:
Can you explain this a bit more? I really like that you split it out from its original form. It really helps clarify the logic.

Reply:
Originally the `parseErrorMessage` checked for an `error` property in the JSON response from CRM, however when I was testing it the property didn't exist, so parseErrorMessage was returning nothing.  The response instead had an `_error` property. I left the check for `json.error` in case there are some other cases where the CRM response does include an `error` property.